diff -u -Nru5 qhull-2020.2/CMakeLists.txt qhull-2020.2.apple-install-name-dir/CMakeLists.txt
--- qhull-2020.2/CMakeLists.txt	2020-09-03 21:23:31.000000000 -0400
+++ qhull-2020.2.apple-install-name-dir/CMakeLists.txt	2021-08-26 16:23:16.604552406 -0400
@@ -368,12 +368,12 @@
     OUTPUT_NAME "${qhull_SHAREDR}$<$<CONFIG:Debug>:d>")
 
 if(UNIX)
     target_link_libraries(${qhull_SHAREDR} m)
     if(APPLE)
-        set_target_properties(${qhull_SHAREDR} PROPERTIES 
-            INSTALL_NAME_DIR "${LIB_INSTALL_DIR}")
+        #set_target_properties(${qhull_SHAREDR} PROPERTIES 
+        #    INSTALL_NAME_DIR "${LIB_INSTALL_DIR}")
     else()
         set_target_properties(${qhull_SHAREDR} PROPERTIES 
             INSTALL_RPATH "${LIB_INSTALL_DIR}"
             INSTALL_RPATH_USE_LINK_PATH TRUE
             BUILD_WITH_INSTALL_RPATH FALSE)
@@ -395,12 +395,12 @@
     OUTPUT_NAME "qhull$<$<CONFIG:Debug>:_d>")
 
 if(UNIX)
     target_link_libraries(${qhull_SHARED} m)
     if(APPLE)
-        set_target_properties(${qhull_SHARED} PROPERTIES 
-            INSTALL_NAME_DIR "${LIB_INSTALL_DIR}")
+        #set_target_properties(${qhull_SHARED} PROPERTIES 
+        #    INSTALL_NAME_DIR "${LIB_INSTALL_DIR}")
     else()
         set_target_properties(${qhull_SHARED} PROPERTIES 
             INSTALL_RPATH "${LIB_INSTALL_DIR}"
             INSTALL_RPATH_USE_LINK_PATH TRUE
             BUILD_WITH_INSTALL_RPATH FALSE)
@@ -423,12 +423,12 @@
     OUTPUT_NAME "${qhull_SHAREDP}$<$<CONFIG:Debug>:d>")
 
 if(UNIX)
     target_link_libraries(${qhull_SHAREDP} m)
     if(APPLE)
-        set_target_properties(${qhull_SHAREDP} PROPERTIES 
-            INSTALL_NAME_DIR "${LIB_INSTALL_DIR}")
+        #set_target_properties(${qhull_SHAREDP} PROPERTIES 
+        #    INSTALL_NAME_DIR "${LIB_INSTALL_DIR}")
     else()
         set_target_properties(${qhull_SHAREDP} PROPERTIES 
             INSTALL_RPATH "${LIB_INSTALL_DIR}"
             INSTALL_RPATH_USE_LINK_PATH TRUE
             BUILD_WITH_INSTALL_RPATH FALSE)
